rototiller.gemspec

```diff
@@ -16,5 +16,7 @@ Gem::Specification.new do |s|
   s.files         = Dir['[A-Z]*[^~]'] + Dir['lib/**/*.rb'] + Dir['spec/*']
 
   #Run time dependencies
-  s.add_runtime_dependency 'rake', '>= 0.9.0'
+  rake_ver = ENV['RAKE_VER'] || '0.9.0'
+  # RAKE_VER=0.9.0, 10.0, 11.0
+  s.add_runtime_dependency 'rake', "~> #{rake_ver}"
```

Comment:
I hate how bundler lets you specify a local copy of a gem if you are using a git reference, but not if you just specify a version.... Anyhow, LGTM.